.claude/commands/analyze.md

```
@@ -0,0 +1,41 @@
+**Purpose**: Multi-dimensional code and system analysis
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)